src/static/js/earn.js

```diff
@@ -1016,197 +2500,241 @@ async function main() {
+                        </div>
+                        <div class="row">
+                            <div class="form-inline w-50 ">
+                                <div class="form-group m-md-0">
+                                    <p class="m-0 font-size-12 font-weight-light">Daily:</p>
+                                    <p class="m-0 font-size-12 font-weight-light">Yearly:</p>
+                                </div>
+                            </div>
+                            <div class="form-inline w-50 mx-auto">
+                                <div class="form-group m-md-0">
+                                <p class="m-0 font-size-12 font-weight-regular">${eDayAPR}% </p>
+                                <p class="m-0 font-size-12 font-weight-regular">${eYearAPR}%</p>
+                                </div>
+                            </div>
+                        </div>
+                    </div>
+                    ${combinedAprDisplay}
+                    <div class="col-sm-12 col-md-2 align-items-center text-center snob-tvl pb-10 pb-md-0">
+                        <p class="m-0 font-size-12"><ion-icon name="bowling-ball-outline"></ion-icon> Allocation</p>
+                        <span class="badge font-size-12 px-5 px-sm-10 mx-5 font-weight-regular">${options.pool_weight *
+                          100}%</span>
+                        <p class="m-0 font-size-12 pt-10"><ion-icon name="ellipse-outline"></ion-icon> SNOB per day</p>
+                        <span class="badge font-size-12 px-5 px-sm-10 mx-5 font-weight-regular">${((snowballsPerBlock *
+                          options.pool_weight) /
+                          1e18) *
+                          15000}</span>
+                    </div>
+                    <div class="col-sm-12 col-md-2 align-items-center d-flex flex-column text-center snob-tvl pb-10 pb-md-0">
+                        <p class="m-0 font-size-12"> Pool Size</p>
+                        ${poolSize}
+                    </div>
+                    ${poolShare}
 
+                </div>
+                <div class="row pt-20">
+                    ${earning}
+                    ${stakeDisplay || ''}
+                    ${availableStake}
+                    <div class="col-sm-12 col-md-2 align-items-center text-center snob-tvl pb-10 pb-md-0">
+                        <p class="m-0 font-size-12"><ion-icon name="flame-outline"></ion-icon> Pending SNOB</p>
+                        <p class="m-0 font-size-16 font-weight-regular">${(options.pending_tokens / 1e18).toFixed(
+                          6
+                        )}</p>
+                    </div>
+                    ${availableUnstake}
+                </div>
+            </div>
+        </div>
+    </div>`
+      $('#snob-pools-used').append(poolPrint)
+    }
   }
-  function poolS3F(options) {
-    var eDayAPR = options.icequeen_apr;
-    var eYearAPR = options.icequeen_apr * 365;
+  function poolS3D(options) {
```

Comment:
This is 2 different functions too. poolS3D starts from line **871** of the red column